src/components/expression/expression-panel.js

```diff
@@ -217,9 +224,9 @@ genotet.ExpressionPanel.prototype.updateGenes = function(gene) {
 /**
  * Adds the cell info into a given container.
  * @param {string} geneName Gene name of which info is to be displayed.
- * @param {string} conditionName Condition name of which info is to be
+ * @param {?|string} conditionName Condition name of which info is to be
```

Comment:
undefined|string, or null|string. Question mark tells the compiler to NOT type-checking at all which is probably not what you want.